src/routes/signup/+page.svelte

```diff
@@ -74,6 +82,33 @@ circumstance.
             onConfirm: onConfirm,
         })
     }
+
+    const connectWallet = async () => {
```

Comment:
This function can be simplified, too. See notes on this function on the login page. The same things hold true about not needing to check registration status.

The main things this function needs to do is get the address from the wallet, store it in the `walletStore`, fund it, and redirect.

Reply:
same here